test/Microsoft.AspNetCore.ApplicationInsights.HostingStartup.Tests/Microsoft.AspNetCore.ApplicationInsights.HostingStartup.Tests.csproj

```diff
@@ -3,8 +3,9 @@
   <Import Project="..\..\build\common.props" />
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp2.0</TargetFramework>
-    <PackageTargetFallback>$(PackageTargetFallback);portable-net40+sl5+win8+wp8+wpa81;portable-net45+win8+wp8+wpa81</PackageTargetFallback>
+    <TargetFrameworks>netcoreapp2.0;net461</TargetFrameworks>
+    <TargetFramework Condition=" '$(OS)' != 'Windows_NT' ">netcoreapp2.0</TargetFramework>
```

Comment:
Wait a second this looks wrong. You are setting the `TargetFrameworks` property but you are not overriding it when it's xplat, you are setting a different property `TargetFramework`.